opentreemap/exporter/tests.py

```diff
@@ -287,39 +287,39 @@ def test_export_users_json_keep_info_private(self):
         commander, user1data, user2data = data
         self.assertFalse('first_name' in user1data)
 
-    def test_export_users_json_make_info_public(self):
-        self.user1.make_info_public = True
-        self.user1.save()
-
-        resp = users_json(make_request(), self.instance)
-
-        data = json.loads(resp.content)
-
-        commander, user1data, user2data = data
-
-        self.assertEquals(commander['username'], self.commander.username)
-        self.assertEquals(user1data.get('email'), None)
-        self.assertUserJSON(user1data,
-                            {'username': self.user1.username,
-                             'email_hash': self.user1.email_hash,
-                             'first_name': self.user1.first_name,
-                             'last_name': self.user1.last_name,
-                             'organization': self.user1.organization,
-                             'allow_email_contact': 'False',
-                             'role': 'commander',
-                             'created': str(self.user1.created)})
-
-        self.assertUserJSON(user2data,
-                            {'last_edit_model': 'Plot',
-                             'last_edit_model_id': str(self.plot.pk),
-                             'last_edit_instance_id': str(self.instance.pk),
-                             'last_edit_user_id': str(self.user1.pk),
-                             'email': 'genly@example.com',
-                             'email_hash': self.user2.email_hash,
-                             'last_edit_model': 'Tree',
-                             'last_edit_model_id': str(self.tree.pk),
-                             'last_edit_instance_id': str(self.instance.pk),
-                             'last_edit_user_id': str(self.user2.pk)})
+#    def test_export_users_json_make_info_public(self):
```

Comment:
The assert in this test does work, but has what looks like some copy/paste or merge cruft.

Python will let you redefine keys when declaring a literal map. The last value wins.

```python
In [1]: d = {'a':1 , 'b': 2, 'a': 3}

In [2]: d
Out[2]: {'a': 3, 'b': 2}
```

The literal hash can be fixed to remove redefined keys, and the test still passes.

```diff
diff --git a/opentreemap/exporter/tests.py b/opentreemap/exporter/tests.py
index c08c3c8..ab9235b 100644
--- a/opentreemap/exporter/tests.py
+++ b/opentreemap/exporter/tests.py
@@ -310,16 +310,12 @@ class UserExportsTest(UserExportsTestCase):
                              'created': str(self.user1.created)})
 
         self.assertUserJSON(user2data,
-                            {'last_edit_model': 'Plot',
-                             'last_edit_model_id': str(self.plot.pk),
-                             'last_edit_instance_id': str(self.instance.pk),
-                             'last_edit_user_id': str(self.user1.pk),
-                             'email': 'genly@example.com',
-                             'email_hash': self.user2.email_hash,
-                             'last_edit_model': 'Tree',
+                            {'last_edit_model': 'Tree',
                              'last_edit_model_id': str(self.tree.pk),
                              'last_edit_instance_id': str(self.instance.pk),
-                             'last_edit_user_id': str(self.user2.pk)})
+                             'last_edit_user_id': str(self.user2.pk),
+                             'email': 'genly@example.com',
+                             'email_hash': self.user2.email_hash})
 
     def test_min_edit_date(self):
         last_week = now() - datetime.timedelta(days=7)
```

I tried both commenting out and fixing this test and it did not resolve the hard to pin down memory pressure issues we are currently running into.

I would prefer that we not include this commit in the PR.